src/ostree/ot-builtin-commit.c

```diff
@@ -96,6 +97,7 @@ static GOptionEntry options[] = {
   { "orphan", 0, 0, G_OPTION_ARG_NONE, &opt_orphan, "Create a commit without writing a ref", NULL },
   { "no-bindings", 0, 0, G_OPTION_ARG_NONE, &opt_no_bindings, "Do not write any ref bindings", NULL },
   { "bind-ref", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_bind_refs, "Add a ref to ref binding commit metadata", "BRANCH" },
+  { "bind-collection-ref", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_bind_collection_refs, "Add a collection–ref binding to commit metadata", "COLLECTION-ID:BRANCH" },
```

Comment:
The new command line options also need to be added to `bash/ostree` (sorry, only just remembered about this).